docs/source/resources/common/exposing.md

```diff
@@ -1,77 +1,124 @@
-# Exposing ScyllaCluster
+# Exposing ScyllaDB cluster
 
-This document explains how ScyllaDB Operator exposes ScyllaClusters in different network setups.
-A ScyllaCluster can be exposed in various network configurations, independently to clients and nodes.
-
-:::{note}
-ScyllaClusters can be only exposed when the ScyllaDB version used version is `>=2023.1` ScyllaDB Enterprise or `>=5.2` ScyllaDB Open Source.
-:::
+This document explains how ScyllaDB Operator exposes ScyllaDB clusters in different network setups.
+A ScyllaDB Cluster can be exposed in various network configurations, independently to clients and nodes.
 
 ## Expose Options
 
 :::{note}
-`exposeOptions` are immutable, they cannot be changed after ScyllaCluster is created.
+`exposeOptions` are immutable, they cannot be changed after ScyllaDB cluster is created.
```

Comment:
nit: a ScyllaDB cluster (since it's no longer a proper noun)
adding articles applies globally

Reply:
done